themes/sirup/sirup.theme.sh

```diff
@@ -15,7 +25,7 @@ function _omb_theme_sirup_rubygem {
  
 function _omb_theme_PROMPT_COMMAND {
   # Check http://github.com/Sirupsen/dotfiles for screenshot
-  PS1="$_omb_prompt_navy\W/$_omb_prompt_bold_navy$(_omb_theme_sirup_rubygem)$_omb_prompt_bold_green$(__git_ps1 " (%s)") ${_omb_prompt_normal}$ "
+  PS1="$_omb_prompt_purple$(omb_theme_syrup_python_venv)$_omb_prompt_navy\W/$_omb_prompt_bold_navy$(_omb_theme_sirup_rubygem)$_omb_prompt_bold_green$(__git_ps1 " (%s)") ${_omb_prompt_normal}$ "
```

Comment:
There seems to be already a better implementation `_omb_prompt_get_python_venv` that also checks `CONDA_DEFAULT_ENV` in addition to `VIRTUAL_ENV`.

https://github.com/ohmybash/oh-my-bash/blob/cb7238fb8f18a2df22b3dfb6f14e35229b5fc56c/lib/omb-prompt-base.sh#L468-L474

`omb-prompt-base.sh` is loaded by default, so the function `_omb_prompt_get_python_venv` is readily available in the theme files:

```suggestion
  local python_venv; _omb_prompt_get_python_venv
  PS1="$_omb_prompt_purple$python_venv$_omb_prompt_navy\W/$_omb_prompt_bold_navy$(_omb_theme_sirup_rubygem)$_omb_prompt_bold_green$(__git_ps1 " (%s)") ${_omb_prompt_normal}$ "
```

Reply:
Some themes like "font" use `_omb_prompt_get_python_venv`. Some themes like "powerline" use `__powerline_python_venv_prompt` defined by itself.
`_omb_prompt_get_python_venv` is only used by garo, font and hawaii50. It's needed to unify all themes.

Reply:
As for the duplication of `__powerline_python_venv_prompt` vs `_omb_prompt_get_python_venv`, I currently think I wouldn't unify it.

- The `powerline` theme in this repository, unlikely the other ones, has its own independent and sophisticated theming system. It's hard to unify the two completely different theming systems (omb-prompt vs powerline) into one. I'm also not sure if there is a motivation to invent a heterogeneous amalgam of two different theming systems.
- I'm not sure which is the exact origin of the theme `powerline` in this repository, but there is likely an upstream somewhere. I wouldn't like to introduce structural changes on the powerline theme independently from the upstream if any. At least, I'd like to be careful before deciding on the large edits on the `powerline` theme.

What I'm thinking for the refactoring is to unify the access to `VIRTUAL_ENV` and other environment variables within a few interfaces, `_omb_prompt_get_python_venv` or similar ones. There are still themes that directly reference `VIRTUAL_ENV` such as `agnoster`, `duru`, `mairan`, `slick` and `zork`, though I haven't decided what to do with each one because I haven't checked the insides of them.

Reply:
Yeah, you are right.

Reply:
ping @juanswan13 Can I apply to the PR branch the change I suggested? If you think there is a reason that we should keep the current version of PR, please let us know!

Reply:
@juanswan13 There seems to be no reply, so I decided to push the change.